filebrowser/base.py

```diff
@@ -219,13 +219,13 @@ def __init__(self, path, site=None):
         else:
             self.path = path
         self.head = os.path.dirname(path)
-        self.filename = os.path.basename(path)
+        self.filename = force_text(os.path.basename(path))
```

Comment:
@sehmaschine  I've found that the problem with `OptionsNamer` was a `''.format(fileobject.filename)`. The `os.path.basename` returns a `str` on python2.7. This fix the problem, but It changes `filename` from `str` to `unicode` in python2.7. 

If you think that this is a bad idea, I can remove it from here and use `force_text` next to the format that I need.